nengo_spinnaker/netlist/objects.py

```diff
@@ -0,0 +1,71 @@
+class Net(object):
```

Comment:
Since this is no longer a subclass nor trivial extension of Rig's "net" concept I'd prefer if this had an alternative name to distinguish it.